sconfig/shared/src/main/scala/org/ekrich/config/impl/SimpleConfig.scala

```diff
@@ -5,19 +5,18 @@ package org.ekrich.config.impl
 
 import java.io.ObjectStreamException
 import java.io.Serializable
-import java.{lang => jl}
+import java.lang as jl
+import java.util as ju
```

Comment:
This is a nit but could you move the `util` import back down after the `time` imports so it is alphabetical?

Reply:
may we add scalafix with
```
OrganizeImports {
  groupedImports = Keep
  importsOrder = Ascii
}
```
for that, please?

Reply:
I am not opposed to scalafix or scalafmt rules but it is too much for me to consider right now. The imports are in pretty good shape with order and groups. Maybe not 100% obvious the groups or maybe not 100% consistent right now.

Reply:
Sure, lets move on with those PRs.
optimizing imports can be done other time